app/publishing/save-request.js

```diff
@@ -3,17 +3,19 @@ const db = require('../data')
 const saveStatement = require('./save-statement')
 const sendCrmMessage = require('../messaging/send-crm-message')
 const createFailure = require('../monitoring/create-failure')
+const saveDelivery = require('./save-delivery')
 
-const saveRequest = async (request, reference, method, reason) => {
+const saveRequest = async (request, reference, method, errorObject) => {
   const transaction = await db.sequelize.transaction()
   try {
     const timestamp = new Date()
     const statement = await saveStatement(request, timestamp, transaction)
     const delivery = await saveDelivery(statement.statementId, method, reference, timestamp, transaction)
-    if (reason) {
-      console.log(`Unable to deliver statement ${statement.filename} to "${statement.email}": ${reason}`)
-      await sendCrmMessage(statement.email, statement.frn, reason)
-      await createFailure(delivery.deliveryId, reason, transaction)
+    
+    if (errorObject.reason) {
```

Comment:
I'm not sure this is right; reason is set to undefined unless the error is triggered by the email validation rule i.e. we might not be handling a rejection from the API itself here.